posts/2020/07/writing-docs-is-not-just-writing-docs.md

```diff
@@ -0,0 +1,39 @@
+
+<!-- TEASER_END -->
+
+Improving Spyder’s documentation started as part of a NumFOCUS Small Development Grant awarded at the end of last year. The goal of the project was not only to update the documentation for Spyder 4, but also to make it more user-friendly, so users can understand Spyder’s key concepts and get started with it more easily.
+ 
+One of the main ideas for this project was to create a series of short video tutorials, explaining the basic functionality of Spyder and its most important panes, allowing users to learn how to use Spyder faster and easier.
+ 
+Carlos Cordoba, our lead maintainer, thought I was the perfect person for this project because of my “good communication and organization skills”, my “clear and fluent English” (his words) and my previous experience at video editing and recording, which I actually gained by recording singing videos during my “YouTuber” phase. 
+ 
+I’ve always been interested in education and worked as a tutor for several years learning different tools, gaining experience on how to teach and questioning the effectiveness of current educational methods. This was the first reason why I got interested in this project. For me, documentation is just a fancy way of saying “educating people on using software”. The challenge here was not recording and editing the videos (which was actually a pretty arduous task), or planning the content for them; the real challenge was to make an impact in such a way that users could find documentation actually useful.
+ 
+When users start to use a new IDE, or any new software, they usually refer to its documentation, which sometimes doesn’t give enough tools for them to start from zero. This was, then, the whole purpose of the tutorial videos. Spyder’s documentation was already very complete in terms of explaining all the features and cool things you can do with each of its panes. However, if I’m a completely new user and I don’t even know how to open it, where do I start? 	
+ 
+I planned these videos as a series of progressive steps that can get users from zero experience to actually working with Spyder. Hence, I divided the videos into three sections, “First Steps with Spyder”, “Working with Spyder”, and “Building Projects with Spyder”. Each section builds on the one before in a way that they are clear enough so that users can find their way through Spyder without knowing anything about it.
+ 
+The [“First Steps with Spyder”](https://www.youtube.com/playlist?list=PLPonohdiDqg9epClEcXoAPUiK0pN5eRoc) section, live now on our YouTube channel, has three videos that provide a starting point for new users before they even open Spyder for the first time. The first tutorial covers different ways of opening Spyder, the basics of using its interface and an introduction to its four main panes, along with a quick look at the others so that users can get familiar with how Spyder is organized. 
```

Comment:
```suggestion
The [“First Steps with Spyder”](https://www.youtube.com/playlist?list=PLPonohdiDqg9epClEcXoAPUiK0pN5eRoc) section, live now on our YouTube channel, has three videos that provide a starting point for new users before they even open Spyder for the first time. The [first tutorial](https://youtu.be/E2Dap5SfXkI) covers different ways of opening Spyder, the basics of using its interface and an introduction to its four main panes, along with a quick look at the others so that users can get familiar with how Spyder is organized. 
```